addons/account/models/account_move_line.py

```diff
@@ -1534,11 +1534,11 @@ def copy_data(self, default=None):
         return data_list
 
     def _search_panel_domain_image(self, field_name, domain, set_count=False, limit=False):
-        if field_name != 'account_root_id' or set_count:
+        query = self._search(domain, limit=1)
+        if not query or field_name != 'account_root_id' or set_count:
```

Comment:
From the screenshot of the traceback what I see is that the type of `query` is a list. The fact that it words for you by checking if it is falsy doesn't seem to be the right fix